docs/apis.md

```diff
@@ -76,6 +76,19 @@ The following schema is used both when retrieving the current configs from the A
 - Normal Response Codes: NoContent(204)
 - Error Response Codes: Unauthorized(401), BadRequest(400)
 
+##### POST Example
```

Comment:
Using a `#####` title will break a bit the page layout when this documented will be rendered in our website. Do you mind replacing it with a text like `The POST request body is expected to be like the following example:`